Monika After Story/game/script-topics.rpy

```diff
@@ -546,6 +546,32 @@ label monika_selfesteem:
             m 3hub "I'll always root for you, [player]!"
     return "derandom"
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="songwriting", 
+            category=["music"],
+            prompt="Songwriting", 
+            random=True, 
+            pool=True
+        )
+    )
+
+label monika_example:
+    m 3eub "Hey [player], have you ever written a song?"
+    m 3ekb "I've tried myself, but it's hard to do with my limited resources here."
+    m 4eub "You know, I used to write some from time to time in my sophmore years. I used to be pretty good."
```

Comment:
I could be wrong on this, but didn't Monika only start learning to play the piano and get into music because she wanted to write a song for you, and had very limited time to do so?

If this is the case, I don't see any other instruments Monika would be playing or writing songs for, and thus, no reason for Monika to write songs in her previous years.

Reply:
I'll double check, as it totally should stay consistent with the story. Might have to redo that then.